integrations/hitl-salesforce/src/actions/hitl.ts

```diff
@@ -56,7 +56,7 @@ export const startHitl: bp.IntegrationProps['actions']['startHitl'] = async ({ c
 
     await salesforceClient.createConversation(newSalesforceConversationId, {
       firstName: (splitName?.length && splitName[0]) || 'Anon',
-      _lastName: !splitName?.length || splitName[splitName.length] || '',
+      _lastName: splitName && splitName?.length > 1 && splitName[splitName.length] || '',
```

Comment:
you don't need the ? if you do splitName && (I think).

Reply:
Yes, I will fix this on my next PR